distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -442,6 +443,64 @@ type Number
      floating point numbers.
 @Builtin_Type
 type Float
+    ## GROUP Conversions
+       ICON convert
+       Generic conversion of an arbitrary Enso value to requested type.
+       Delegates to appropriate `.from` conversion method, if it exists.
+       If such method doesn't exist, `No_Such_Conversion` panic is raised.
+
+       Arguments:
+       - target_type: the requested type.
+
+       > Example
+         Following code defines conversion of a `Complex` type to a `Number`
+         by computing absolute distance from `0`. The code yields `5.0`:
+
+             type Complex
+                 Value re:Number im:Number
+
+             Number.from (that:Complex) = that.re*that.re+that.im*that.im . sqrt
+
+             Complex.Value 3 4 . to Number
+
+       ? Creating `Decimal`s and Converting to `Decimal`
+
+         When creating a `Decimal` from a literal floating-point value, the
+         preferred method is to express the literal as a string and use
+         `Decimal.from_text`, since this will give a `Decimal` that matches the
+         value precisely.
+
+         To convert a `Float` or `Integer` to a `Decimal`, use its `.to_decimal`
+         method. You can also use `Float.to Decimal` to convert a `Float` to a
+         `Decimal`.
```

Comment:
It feels like this is still missing information about when I should expect a warning and when not.

Reply:
This is removed. The same documentation exists in Decimal.enso but there it is followed by error/warning documentation.